Formula/f/foot.rb

```diff
@@ -0,0 +1,62 @@
+class Foot < Formula
+  desc "Fast, lightweight and minimalistic Wayland terminal emulator"
+  homepage "https://codeberg.org/dnkl/foot"
+  url "https://codeberg.org/dnkl/foot/archive/1.18.1.tar.gz"
+  sha256 "59d22187f7ceaaaa570a5299b102e8f4692826e98785f89ad9d8911802ccc000"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "scdoc" => :build
+
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "libxkbcommon"
+  depends_on :linux
+  depends_on "pixman"
+  depends_on "wayland"
+
+  def install
+    ENV["CFLAGS"] = "#{ENV["CFLAGS"]} -O3"
```

Comment:
```suggestion
    ENV.O3
```
Is there a reason why we shouldn't be using Homebrew's default optimisation level here?

Reply:
It was recommended by the author for GCC 10, which doesn't apply anymore, so I removed it.